ethereum/blockwatch/block_watcher.go

```diff
@@ -171,7 +171,7 @@ func (w *Watcher) Watch(ctx context.Context) error {
 			// return an error.
 			return err
 		}
-		log.WithError(err).Error("blockwatch.Watcher error encountered")
+		log.WithError(err).Warn("blockwatch.Watcher error encountered")
```

Comment:
Is it really okay to demote all block watcher errors to warnings? Aren't these sometimes an indication of a bug that we need to address?

Reply:
The errors we see in Kibana are:
- `unknown block`
- `not found`
- `couldn't parse parameters: blockhash`
- `Post https://...: context deadline exceeded`

The only one that _could_ indicate a bug or issue is `couldn't parse parameters: blockhash`. I'll special case the other errors instead of blanked using `Warn`.